src/Perplex.ContentBlocks/App_Plugins/Perplex.ContentBlocks/perplex.content-blocks.html

```diff
@@ -66,6 +73,9 @@ <h3>
                             </div>
                         </div>
                     </div>
+                    <div ng-if="filteredResults.length < 1">
+                        <localize key="perplexContentBlocks_noContentBlocksMatchedYourSearch">No Content Blocks matched your search with</localize><span> "{{vm.state.ui.picker.search}}".</span>
+                    </div>
```

Comment:
This message is now displayed in a grid cell so will not have much space available, is that intended?
![msg-grid](https://user-images.githubusercontent.com/24297947/103879115-b3d1e480-50d7-11eb-8f8e-0912b082c90b.png)

I think it's better to move this message to the `<div class="p-col--1-1">` that holds the input and add a bit of `margin-top`:
![msg](https://user-images.githubusercontent.com/24297947/103879187-cfd58600-50d7-11eb-9952-a6b482f7ccc5.png)

What do you think?

Reply:
Agree! I did not test with such a long value ;). Will add it in my next commit.